workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/DealRegistrationList/index.css

```diff
@@ -0,0 +1,30 @@
+div:has(> table.custom-table) {
```

Comment:
@abfranca @thaynam do these css files get automatically namespaced or applied inline? If not then we should make this more specific as to avoid the styles bleeding onto all the other elements on the page

Reply:
@ryanschuhler this "custon-table" class is been passed as a prop to the table component, so it wont interfere in other tables that don't have this class. Is that what you asked? Shall we use a more specific name for this new class?

Reply:
@abfranca I was more concered about the rest of the css, sorry that was unclear. I guess for this instance mainly the 'th' is being globally applied

Reply:
@ryanschuhler oh, i see... i'll reduce the scope for this. thanks.

Reply:
Done!